conf/docker_development.ini

```diff
@@ -0,0 +1,92 @@
+###
```

Comment:
This file is currently identical to the production file. Keeping it for future changes?

Reply:
Yes, I though that we can make changes to the production and dev version independent of each other. But can also delete it?!

Reply:
OK, let's keep it 👍